tests/console/suseconnect.pm

```diff
@@ -0,0 +1,44 @@
+# Summary: 
+# Maintainer: Juraj Hura <jhura@suse.com>
+
+use base "basetest";
+use strict;
+use testapi;
+use utils 'zypper_call';
+use registration;
+use version_utils 'is_sle';
+
+sub run {
+    my $reg_code = get_required_var("SCC_REGCODE");
+    my $arch = get_required_var("ARCH");
+    my $live_reg_code = get_required_var("SCC_REGCODE_LIVE");
+
+    select_console 'root-console';
+    
+    # Make sure to start with de-registered system. In case the system is not registered this command will fail
+    assert_script_run "SUSEConnect -d ; echo 1";
```

Comment:
Why the "echo 1"? A more common way to explicitly ignore the exit code would be `SUSEConnect -d ||:`

Reply:
Thanks for this, will change it to your suggestion.